code/go/0chain.net/smartcontract/zcnsc/nodes_reader.go

```diff
@@ -85,13 +83,7 @@ func GetGlobalNode(ctx cstate.StateContextI) (*GlobalNode, error) {
 		return nil, err
 	}
 
-	gn.MinMintAmount = state.Balance(config.SmartContractConfig.GetInt("smart_contracts.zcn.min_mint_amount"))
-	gn.PercentAuthorizers = config.SmartContractConfig.GetFloat64("smart_contracts.zcn.percent_authorizers")
-	gn.MinAuthorizers = config.SmartContractConfig.GetInt64("smart_contracts.zcn.min_authorizers")
-	gn.MinBurnAmount = config.SmartContractConfig.GetInt64("smart_contracts.zcn.min_burn_amount")
-	gn.MinStakeAmount = config.SmartContractConfig.GetInt64("smart_contracts.zcn.min_stake_amount")
-	gn.BurnAddress = config.SmartContractConfig.GetString("smart_contracts.zcn.burn_address")
-	gn.MaxFee = config.SmartContractConfig.GetInt64("smart_contracts.zcn.max_fee")
+	gn.Config = loadSettings()
```

Comment:
the config saved on  MPT will not be used if we reset back to local settings

Reply:
fixed